torch/nested/_internal/sdpa.py

```diff
@@ -624,6 +639,7 @@ def jagged_scaled_dot_product_attention(
     is_causal=False,
     scale=None,
 ):
+    query, key, value = _autocast(query, key, value)
```

Comment:
Could you add a short comment of why we need this?

Reply:
added one in `_autocast()`